packages/common/src/utils/sagaHelpers.ts

```diff
@@ -14,10 +15,18 @@ import {
 } from 'typed-redux-saga'
 import { Action } from 'typesafe-actions'
 
+import { ErrorLevel, ReportToSentryArgs } from '~/models'
 import { waitForReachability } from '~/store/reachability/sagas'
+import { toast } from '~/store/ui/toast/slice'
 
 import { Status } from '../models/Status'
 
+import { isResponseError } from './error'
+
+const messages = {
+  somethingWrong: 'Something went wrong'
```

Comment:
Is there a chance we end up on the error splash screen _and_ throwing a toast with the same message? My guess is no since landing on the splash would be a result of catching the error before the saga dies. But I could be wrong.

Reply:
I think it's not possible, but maybe can merge this and play around a bunch on staging.audius.co as a group, before doing any mobile work too